### ISSUE TITLE
unicorn ignoring config when programmatically creating items or publishing

### ISSUE THREAD

**Issue by tommyhawk520** [kamsar/Unicorn#33]
I have unicorn 2.0.4 set up in sitecore 7.2, the config is standard but i have added a few extra include and exclude rules. unicorn seems to ignore the exclude rules when i publish an item or create it programmatically. if i just create an item in content editor it doesnt get created

**Comment by kamsar** [comment]
I've retested the exclusion part of this issue and am unable to reproduce it in several different scenario tries. It's worth noting that Sitecore 8 defaults to having schedule items in the database like @RaYell mentioned above that change their last run field. I've got those excluded in several production or in-development Sitecore 8 sites without issue.

I've also tried programmatically creating items in excluded locations:
	
        using (new SecurityDisabler())
	    {
		  var db = Factory.GetDatabase("master");

		  var item = db.GetItem("/sitecore/system/publishing targets");

		  item.Add("cats", new TemplateID(TemplateIDs.Alias));
	    }

No items were written either time.

I'll see if I can replicate the publishing piece.

**Comment by kamsar** [comment]
Ok I tested using the exact publishing code from two comments ago, and I can't replicate that issue either. I tried publishing both with a root item that is excluded and one that is included, and no web database serialization occurred.

My best guess is that there's some kind of event handler, pipeline processor, etc that is doing something unexpected to the way Sitecore works. But without reproduction, I'm going to close this for now.
